src/auth/authoptions.ts

```diff
@@ -148,7 +150,7 @@ export const authOptions: AuthOptions = {
             const userId = user ? Number(user.id) : token?.user.id
 
             // TODO - refactor when read user action exists
-            const userInfo = await prisma.user.findUnique({
+            const userInfo = await prisma.user.findUniqueOrThrow({
```

Comment:
This should prob. be inside prismaCall. We need to go through this at some point